intro-HPC/ch_quick_reference_guide.tex

```diff
@@ -1,52 +1,63 @@
+Setup ftp session & \texttt{sftp \userid{}@\loginnode{}} \\ \hline
 \end{tabular}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Modules}} \\ \hline
-List all available modules & module av \\ \hline
-List loaded modules        & module list \\ \hline
-Load module                & module load $<$name$>$ \\ \hline
-Unload module              & module unload $<$name$>$ \\ \hline
-Unload all modules         & module purge \\ \hline
-Help on use of module      & module help \\ \hline
+List all available modules & \texttt{module avail} \\ \hline
+List loaded modules        & \texttt{module list} \\ \hline
+Load module                & \texttt{module load example} \\ \hline
+Unload module              & \texttt{module unloade example} \\ \hline
+Unload all modules         & \texttt{module purge} \\ \hline
+Help on use of module      & \texttt{module help} \\ \hline
 \end{tabular}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Jobs}} \\ \hline
-Submit Job              & qsub $<$script.pbs$>$ \\ \hline
-Status of the job       & qstat $<$jobid$>$ \\ \hline
-Possible start time (not available everywhere)    & showstart $<$jobid$>$ \\ \hline
-Check job (not available everywhere)              & checkjob $<$jobid$>$ \\ \hline
-Show compute node       & qstat -n $<$jobid$>$ \\ \hline
-Delete job              & qdel $<$jobid$>$ \\ \hline
-Status of all your jobs & qstat \\ \hline
-Show all jobs on  queue (not available everywhere) & showq \\ \hline
+Submit Job              & \texttt{qsub script.pbs} \\ \hline
+Status of the job       & \texttt{qstat jobid} \\ \hline
+\ifgent\else % these are not enabled in ghent
+Possible start time (not available everywhere)    & \texttt{showstart jobid} \\ \hline
+Check job (not available everywhere)              & \texttt{checkjob jobid} \\ \hline
+\fi
+Show compute node       & \texttt{qstat -n jobid} \\ \hline
+Delete job              & \texttt{qdel jobid} \\ \hline
+Status of all your jobs & \texttt{qstat} \\ \hline
+\ifgent\else % these are not enabled in ghent
+Show all jobs on  queue (not available everywhere) & \texttt{showq} \\ \hline
+\fi
 %Show queue summary      & qstat -q \\ \hline
-%Show some queue         & qstat -f -Q $<$queue$>$ \\ \hline
-Submit Interactive job  & qsub -I \\ \hline
+%Show some queue         & qstat -f -Q <{}queue>{} \\ \hline
+Submit Interactive job  & \texttt{qsub -I} \\ \hline
 \end{tabular}
 
 \begin{tabular}{|l|l|} \hline
 \multicolumn{2}{|c|}{\strong{Disk quota}} \\ \hline
-Check your disk quota & mmlsquota \\ \hline
-Check disk quota nice & show\_quota.py \\ \hline
-Local disk usage      & du -h . \\ \hline
-Overall disk usage    & df -a \\ \hline
+\ifgent
+Check your disk quota & See \url{https://account.vscentrum.be} \\ \hline
+\else
+Check your disk quota & \texttt{mmlsquota} \\ \hline
+Check disk quota nice & \texttt{show\_quota.py} \\ \hline
+\fi
+Local disk usage      & \texttt{du -h} . \\ \hline
+Overall disk usage    & \texttt{df -a} \\ \hline
```

Comment:
`-ah` (`h` is for human readable), but this spits out a whole bunch of irrelevant stuff, just use `df -h`?